docs/controls/InfiniteCanvas.md

```diff
@@ -0,0 +1,82 @@
+---
+title: InfiniteCanvas XAML Control
+author: IbraheemOsama
+description: InfiniteCanvas is a canvas that supports Infinite Scrolling, Ink, Text, Format Text, Zoom in/out, Redo, Undo, Export canvas data, Import canvas data.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, InfiniteCanvas, XAML Control, xaml
+---
+
+# InfiniteCanvas XAML Control
+
+The [InfiniteCanvas Control](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.controls.infinitecanvas) is a canvas that supports Infinite Scrolling, Ink, Text, Format Text, Zoom in/out, Redo, Undo, Export canvas data, Import canvas data.
+
+## Syntax
+
+```xaml
+<Page ...
+     xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"/>
+
+<controls:InfiniteCanvas Name="canvas"/>
```

Comment:
Add some Properties and Events in this syntax. This look .....

Reply:
The default usage for the canvas is "no properties" that's why it is better this way :)